src/components/Layout.jsx

```diff
@@ -72,7 +72,7 @@ const Layout = ({ children, langCode }) => {
           <SkipLink href="#main">
             {getTranslation(langCode, "header.skiplink")}
           </SkipLink>
-          <Link to="/">
+          <Link to={langCode === "en" ? "/" : `/${langCode}`}>
```

Comment:
I have a small request to make the code to be more readable; Could you move this into a variable, that gets passed to the `to`-attribute? So something like 

```
const linkToIndex = langCode === "en" ? "/" : `/${langCode}`
```
and then
```
<Link to={linkToIndex}>
```

Reply:
okaaaay, on it

Reply:
hi @eevajonnapanula i already fix it as your code review